.github/workflows/ci.yml

```
@@ -120,6 +120,9 @@ jobs:
       - name: Push Docker image
         run: make docker-push
 
+      - name: Tag Docker image with dev-latest
+        run: make docker-tag
```

Sound a bit misleading, the DOCKER_TAG is made elsewhere. We are pushing the dev-latest there.
```suggestion
        run: make docker-push-dev-latest
```

---

Well actually it perform both tasks, 'tagging' and 'pushing' in one target inside the makefile. So not 100% sure if the naming should be `docker-tag-push-dev-latest`